### ISSUE TITLE
Naming an headless output makes clients detect it twice

### ISSUE THREAD

**Issue by pcouy** [hyprwm/Hyprland#6901]
### Regression?

Yes, No

### System Info and Version

<details> <summary>System/Version info</summary>

```sh
<Paste the output of the command here>
```

</details>

### Description

After upgrading and trying to name an headless output (`hyprctl output create headless RPI`), I noticed waybar appearing twice on this output. Inspecting the output of waybar I found the attached log extract.

hyprpaper logged a duplicated line as well for this named headless output ("Image data for RPI: [...]")

After tracking the probable cause to 82099fd, I tried running `hyprctl output create headless` with no name, and dealing with the auto-generated `HEADLESS-*` name instead. This resolved the issue.

### How to reproduce

Add the following line to your `hyprland.conf` : 

```
monitor=test,1920x1080@30, 1920x0, 1
```

Then run the following commands : 
```
hyprctl output create headless test
waybar # Take a screenshot of `test` output to confirm the issue
hyprpaper # (Don't forget to define a WP for the `test` output) Notice the wallpaper being applied twice in stdout
```

### Crash reports, logs, images, videos

```
[2024-07-16 17:34:22.190] [info] Bar configured (width: 1910, height: 30) for output: eDP-1
[2024-07-16 17:34:22.190] [info] Bar configured (width: 1910, height: 30) for output: HDMI-A-1
[2024-07-16 17:34:22.190] [info] Bar configured (width: 1190, height: 30) for output: RPI
[2024-07-16 17:34:22.190] [info] Bar configured (width: 1190, height: 30) for output: RPI
```

**Comment by pcouy** [comment]
Is there anything wrong with this issue ? Do I need to add more details ?

**Comment by vaxerski** [comment]
no, I don't have anything to say. I have 900 open issues, and this isnt a pressing issue at all
